store/datastore.go

```diff
@@ -33,6 +34,8 @@ type DataStore interface {
 	UpsertDeviceStatus(ctx context.Context, tenantID, deviceID, status string) error
 	AllocateSession(ctx context.Context, sess *model.Session) error
 	GetSession(ctx context.Context, sessionID string) (*model.Session, error)
+	GetSessionRecording(ctx context.Context, sessionID string, w io.Writer) error
+	SetSessionRecording(ctx context.Context, sessionID string, sessionBytes []byte) error
```

Comment:
Nitpick: "Set" sounds like an update/replace operation.
```suggestion
	InsertSessionRecording(ctx context.Context, sessionID string, sessionBytes []byte) error
```

Reply:
sure!